app/components/trainees/confirmation/diversity/view.rb

```diff
@@ -6,23 +6,27 @@ module Diversity
       class View < GovukComponent::Base
         include SanitizeHelper
 
-        attr_accessor :trainee
+        attr_accessor :data_model
 
-        def initialize(trainee:)
-          @trainee = trainee
+        def initialize(data_model:)
```

Comment:
If it's just the one arg, you could remove the keyword arg to shave some keystrokes

Reply:
I happy to do this after all the sections are done. Fow now, I suggest leaving it because we have other views with the same signature.